activities/finance-management/bookkeeping.md

```diff
@@ -0,0 +1,57 @@
+
+## WeFact
+
+WeFact is a lightweight and easy to use program for invoicing. We use it for membership administration. It’s integrated with both Exact as well as Visionplanner. The Operations Coordinator has access to WeFact and creates accounts for other staff members that need access. WeFact has our customized template for our membership payment note. Payment requests are sent directly from WeFact to our members at the beginning of each year from our own domain via <info@publiccode.net>.
+
+## Payments
+
+All payments are managed by the Operation Coordinator. The Operations Coordinator makes sure there is always an underlying document or receipt (Boekstuk) for each transaction. Having a papertrail is essential.
+
+## Invoices – payable
+
+The Operations Coordinator is responsible for paying and processing all the invoices and payment requests that come in. The process goes as follows:
+
+* Send paid invoices to <2y4k@inkoop.exactonline.nl> as PDF, including expense reports from third parties
+* Make sure to add right ledger in the title of the PDF so the bookkeeper knows where to book it
+* Do NOT send invoices from Rabobank (booked from Bank), GetMonkeys, and Tentoo, as these are all integrated with Exact
+* Do NOT send receipts from underlying expense forms, as this would lead to duplication
+* Filing: The Operations Coordinator keeps invoices and expense forms organized by month and year in the [Secret Finance Drive.](https://drive.google.com/drive/folders/1B47cfi314HJzoW7_QDaaaG1hfvxwCjpJ)
+
+## Invoices – receivable
+
+* Send to <2y4k@verkoop.exactonline.nl> as PDF.
+
+## Bank statements
+
+* Bank statements can be downloaded from the Rabobank dashboard
+* Rabobank is linked to Exact, so we don't need to submit the statements
+* Credit card statements are send by snail-mail as well as available in the Rabobank dashboard. The credit card statements need to be mailed to Exact every month: <2y4k@verkoop.exactonline.nl> as PDF.
+
+## Budget
+
+* The budget blueprint (spreadsheet) which is used as the baseline for budgeting and forecasting was constructed with help of our finance consultant, Rob Schouten.
+* The budget is approved by the board by the end of each year and is added to Visionplanner
+* The Operations Coordinator is responsible for budgeting and adding it to Visionplanner
+
+## Tax status pertaining financials
+
+* We have ANBI status: read more information about ANBI benefits and requirements
```

Comment:
Link is missing to the "Read more information about ANBI benefits and requirements"